File: android/app/src/main/kotlin/com/wrbug/gitbbs/flutterbridge/mmkv/method/GetTokenCall.kt
package com.wrbug.gitbbs.flutterbridge.mmkv.method

import com.wrbug.gitbbs.flutterbridge.FlutterMethodCall
import com.wrbug.gitbbs.mmkv.UserKv
import io.flutter.plugin.common.MethodCall
import io.flutter.plugin.common.MethodChannel


class GetTokenCall : FlutterMethodCall {
    override fun invoke(call: MethodCall, result: MethodChannel.Result) {
        result.success(UserKv.getToken())
    }


    override fun getMethodName() = "getToken"
}

File: ios/Runner/FlutterBridge/mmkv/method/GetTokenCall.swift
//
//  GetTokenCall.swift
//  Runner
//
//  Created by WrBug on 2019/3/7.
//  Copyright © 2019 The Chromium Authors. All rights reserved.
//

import Foundation
class GetTokenCall: MethodCall {
    func getMethodName() -> String {
        return "getToken"
    }
    
     func invoke(call: FlutterMethodCall, result: (Any?) -> Void) {
        result(UserKv.getToken())
    }
}
